src/app/globals.css

```
@@ -2,37 +2,26 @@
-  font-weight: 400;
-  font-style: normal;
-}
-
-@font-face {
-  font-family: "BitBit";
-  font-style: normal;
-  font-weight: 400;
-  src: url("//cdn.df.nexon.com/img/common/font/DNFBitBitv2.otf")
-    format("opentype");
-}
+@import "react-toastify/dist/ReactToastify.css";
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)